app/src/main/java/ch/epfl/sweng/studyup/questions/DisplayQuestionActivity.java

```diff
@@ -0,0 +1,126 @@
+package ch.epfl.sweng.studyup.questions;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import ch.epfl.sweng.studyup.MainActivity;
+import ch.epfl.sweng.studyup.R;
+import ch.epfl.sweng.studyup.player.Player;
+
+public class DisplayQuestionActivity extends AppCompatActivity {
+
+    private final String TAG = "DisplayQuestionActivity";
+    public static final String DISPLAY_QUESTION_URI = "display_question_uri";
+    public static final String DISPLAY_QUESTION_TRUE_FALSE = "display_question_true_false";
+    public static final String DISPLAY_QUESTION_ANSWER = "display_question_answer";
+    public static final int XP_GAINED_WITH_QUESTION = 10;
+
```

Comment:
Shouldn't it be more logical to have those constant in Utils? Or make them private if only this class will use it?